ffi/framebuffer_SDL2_0.lua

```diff
@@ -29,6 +29,17 @@ end
 function framebuffer:resize(w, h)
     w = w or SDL.w
     h = h or SDL.h
+    local output_w = ffi.new("int[1]", 0)
+    local output_h = ffi.new("int[1]", 0)
+    if SDL.SDL.SDL_GetRendererOutputSize(SDL.renderer, output_w, output_h) == 0 and tonumber(output_w[0]) ~= w then
```

Comment:
I assume the double SDL is an unfortunate side-effect of our own ffi wrapper being named SDL, and the C namespace for the actual SDL lib being named SDL too?

Reply:
Yes, we could change one of them if desired for clarity. The construct occurs a fair bit throughout the code dealing with SDL I'm sure. ;-)

The one called `S` in `SDL2_0.lua` is called `SDL` in framebuffer here.
https://github.com/koreader/koreader-base/blob/df96630c21ff9c5ededa1a864da2c048e4c542ed/ffi/SDL2_0.lua#L61-L67